tw-tasks-core/src/main/java/com/transferwise/tasks/TasksProperties.java

```diff
@@ -293,6 +296,29 @@
 
   private Environment environment = new Environment();
 
+  @Valid
+  private Triggering triggering = new Triggering();
+
+  @Data
+  @Validated
+  public static class Triggering {
+
+    @Valid
+    private Kafka kafka = new Kafka();
+
+    @Data
+    @Validated
+    public static class Kafka {
+
+      @NotBlank
+      @ResolvedValueConstraint
+      private String bootStrapServers;
+
+      private Map<String, String> properties = new HashMap<>();
```

Comment:
A doc explaining that these properties override the default ones might be useful 🤔